quarterback-packages/api/rollup.config.js

```diff
@@ -27,6 +27,9 @@ export default {
     {
       file: pkg.module,
       format: 'es',
+      globals: {
```

Comment:
@mbetamony this is related to making @manuscripts/transform work in the api.

Reply:
I see in your build2 & dev2 you stopped using rollup, however if we were to continue using it did this make the window  defined for the transformer? I've tried multiple ways but non seem to work for some reason.

Reply:
The main reason why I stopped using rollup is because I was suspecting it wasn't doing proper tree-shaking of the transformer components and that I didn't need any DOM solutions like these one, because the error of not finding window and the document was in the code we never execute on frontend. Later I had a look at pressroom and find out that it is also a problem there so I borrowed the fix from there.
I don't think that this config does anyhting. I provide DOM api in quarterback-packages/api/src/define-globals.ts